alt1/scripts/ws.ts

```diff
@@ -175,7 +176,10 @@ export class WebSocketClient {
 
     processEvent(eventData: EventRecord): void {
         console.log("Parsed data: ", eventData);
-        if (eventData.type === "addEvent") addNewEvent(eventData);
+        if (eventData.type === "addEvent") {
+            addNewEvent(eventData);
+            notifyEvent(eventData);
+        }
         if (eventData.type === "editEvent") updateEvent(eventData);
         if (eventData.type === "deleteEvent") removeEvent(eventData);
```

Comment:
Would be good to add the same for edits and deletions. Should be able to still use `notifyEvent` mostly.